src/scrubber.ts

```diff
@@ -16,18 +17,30 @@ export class Scrubber {
     this.initializationVector = initialzationVector ? initialzationVector : nanoid()
     this.scrubbers = { ...defaultScrubbers, ...additionalScrubbersImpl }
     this.cfg = { ...defaultCfg, ...this.expandCfg(cfg) }
+    this.cfg.splitFields = this.splitFields(cfg)
     this.checkIfScrubbersExistAndRaise(cfg, this.scrubbers)
   }
 
   scrub<T>(data: T): T {
     return this.applyScrubbers(data)
   }
 
-  private applyScrubbers<T>(data: T): T {
+  private applyScrubbers<T>(data: T, parents?: string[]): T {
     const dataCopy = Array.isArray(data) ? [...data] : { ...data }
 
     Object.keys(dataCopy).forEach(key => {
-      const scrubberCurrentField = this.cfg.fields[key]
+      let scrubberCurrentField = this.cfg.fields[key]
+
+      if (
+        !scrubberCurrentField &&
+        this.cfg.splitFields &&
+        this.cfg.splitFields[key] &&
```

Comment:
FYI: there's this syntax: `this.cfg.splitFields?.[key]`

Reply:
Basically `?.` is a "safe access operator"

Reply:
Can be applied to a function call to, e.g `some.function?.()`

Reply:
Nice, I did not know it could be applied to calls to indices.